x-pack/legacy/plugins/apm/public/components/shared/TransactionActionMenu/__test__/sections.test.ts

```diff
@@ -0,0 +1,197 @@
+              label: 'View sample document',
+              href:
+                '#/discover?_g=(refreshInterval:(pause:true,value:\'0\'),time:(from:now-24h,to:now))&_a=(index:apm_static_index_pattern_id,interval:auto,query:(language:kuery,query:\'processor.event:"transaction" AND transaction.id:"123" AND trace.id:"123"\'))',
+              condition: true
+            }
+          ]
+        }
+      ]
+    ]);
+  });
+
+  it('shows pod and required sections only', () => {
+    const transaction = ({
+      kubernetes: { pod: { uid: '123' } },
+      timestamp,
+      trace: { id: '123' },
+      transaction: { id: '123' }
+    } as unknown) as Transaction;
+    expect(
+      getSections({
+        transaction,
+        basePath,
+        location: ({} as unknown) as Location,
+        urlParams: {}
+      })
+    ).toEqual([
+      [
+        {
+          key: 'podDetails',
+          title: 'Pod details',
+          subtitle: 'View logs and metrics of this pod to get further details.',
+          actions: [
+            {
+              key: 'podLogs',
+              label: 'Pod logs',
+              href: '#/link-to/pod-logs/123?time=1580983200',
+              condition: true
+            },
+            {
+              key: 'podMetrics',
+              label: 'Pod metrics',
+              href: '#/link-to/pod-detail/123?from=&to=',
+              condition: true
+            }
+          ]
+        },
+        {
+          key: 'traceDetails',
+          title: 'Trace details',
+          subtitle: 'View the trace logs to get further details.',
+          actions: [
+            {
+              key: 'traceLogs',
+              label: 'Trace logs',
+              href:
+                '#/link-to/logs?time=1580983200&filter=trace.id:%22123%22%20OR%20123',
+              condition: true
+            }
+          ]
+        }
+      ],
+      [
+        {
+          key: 'kibana',
+          actions: [
+            {
+              key: 'sampleDocument',
+              label: 'View sample document',
+              href:
+                '#/discover?_g=(refreshInterval:(pause:true,value:\'0\'),time:(from:now-24h,to:now))&_a=(index:apm_static_index_pattern_id,interval:auto,query:(language:kuery,query:\'processor.event:"transaction" AND transaction.id:"123" AND trace.id:"123"\'))',
+              condition: true
+            }
+          ]
+        }
+      ]
+    ]);
+  });
+
+  it('shows host and required sections only', () => {
+    const transaction = ({
+      host: { hostname: 'foo' },
+      timestamp,
+      trace: { id: '123' },
+      transaction: { id: '123' }
+    } as unknown) as Transaction;
+    expect(
+      getSections({
+        transaction,
+        basePath,
+        location: ({} as unknown) as Location,
+        urlParams: {}
+      })
+    ).toEqual([
+      [
+        {
+          key: 'hostDetails',
+          title: 'Host details',
+          subtitle: 'View host logs to get further details.',
```

Comment:
Shouldn't this be
```suggestion
          subtitle: 'View host logs and metrics to get further details.',
```

Reply:
The section has a link to either logs or metrics apps

Reply:
Maybe it's in another place then. On the screenshot for this PR there are two headers each with two links (for logs and metrics).

The text for the first:
"View logs and metrics of this container to get further details"

The text for the second:
"View host logs to get further details"

The second text seems inconsistent with the first. Shouldn't it be:
"View logs and metrics of this host to get further details"

Additionally I'd change "of this" to "for this {host/container}"

This screenshot:
![image](https://user-images.githubusercontent.com/209966/74236959-b0b95300-4cd2-11ea-96ea-88dc2b113577.png)

Reply:
Yeah, that makes sense.  I will update it to "... logs and metrics for this ....".